recipes/smallgenomeutilities/0.2/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "smallgenomeutilities" %}
+{% set version = "0.2" %}
+{% set md5 = "f8b222892e9f18bb135efd5802a878a4" %}
+{% set sha256 = "802ca5f85a14e8243aa44e5f4fb771691f0c80689cf3ecb1edce5382ce0ecda3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/f8/e2/54b075294a3247595abf9f7d7314eddc04c77a049398157514b65087fe6a/{{ name|lower }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # not 'noarch python', because of skip version, and because of command line utils
+  skip: True # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - biopython
+    - numpy
+    - progress
+    - pysam
+    - scikit-learn # sklearn
```

Comment:
this comment should be removed

Reply:
PyPI dependencies fixed upstream by @SoapZA, comment no longer relevant.